docs/2.8.0/docs/canton/usermanual/apis.rst

```diff
@@ -212,6 +212,31 @@ using the custom target audience configuration option:
 
 .. literalinclude:: /canton/includes/mirrored/community/app/src/test/resources/documentation-snippets/ledger-api-target-audience.conf
 
+Ledger API Caches
+^^^^^^^^^^^^^^^^^
+
+The ``max-contract-state-cache-size`` and ``max-contract-key-state-cache-size`` parameters control the sizes of the
+ledger api contract and contract key caches respectively. Modifying these parameters changes the likelihood that a
+transaction using a contract or a contract-key that was recently accessed (created or read) can still find it in the
+memory rather than need to query it from the database. Larger caches might be of interest when there is a big pool of
+ambient contracts that are consistently being fetched or used for non consuming exercises. It may also benefit those
+use cases where a big pool of contracts is being rotated through a create -> archive -> create-successor cycle.
+Consider adjusting these parameters explicitly if the performance of your specific workflow depends on large caches.
```

Comment:
Overall this could benefit from using simpler, more direct language and shorter sentences. (We have some long-term editorial plans so will tackle that kind of review later.)